keyboards/hotdox76v2/info.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
No layout macros?

Reply:
source fork didn't have one, was being lazy.  Added, and verified that it's correct, codewise.